server/src/main/java/org/elasticsearch/action/support/SubscribableListener.java

```diff
@@ -191,6 +191,10 @@ public final void addListener(ActionListener<T> listener) {
      *                      <li>Pass the desired executor in as {@code executor}, and</li>
```

Comment:
This line is unclear. I see elsewhere comments say `Ensure that this {@link SubscribableListener} is always completed using that executor, and` -- that would be helpful to update here and L455, IIUC.

Reply:
The `Ensure that this {@link SubscribableListener} is always completed using that executor` only applies for the overloads that don't take their own `Executor` argument.